src/main/java/com/project/psk/model/dto/User/UserUpdateDTO.java

```diff
@@ -0,0 +1,16 @@
+package com.project.psk.model.dto.User;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+//this dto is used in updating
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserUpdateDTO {
```

Comment:
Something to think about. 
Idk if it's necessary to have a separate dto from UserInfoDto as this only emits email. I assume it's so a user can't modify email? If that is the case, I think frontend could just not allow the user to enter email data. Frontend will (probably? because it's not implemented yet xdd) have current user's info, so it could set email automatically to the current user's and send the body with it. Or you could simply not alter email on the service layer even if you get a different email data.

Reply:
yea, i guess it's not really worth having and is kind of confusing, will remove